packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -1,16 +1,17 @@
 import asyncStates from '@zooniverse/async-states'
-import * as d3 from 'd3'
+import { extent } from 'd3'
 import { zip } from 'lodash'
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
-import request from 'superagent'
```

Comment:
Goodbye, `superagent`!

I was going to ask if we shouldn't also remove this from package.json, then I realised that DataImageViwer(?), SingleTextViewer(?), ScatterPlotViewer, and VariableStarViewer still uses the dependency, dangit. I didn't even realise those first two viewers existed.

Reply:
`SIngleTextViewer` inspired me to write this PR, when I saw that it was still using outdated code from when we first started the monorepo.